content/guides/accessibility/color-considerations.mdx

```diff
@@ -0,0 +1,332 @@
+
+A color value that is APCA-compliant **might not** be WCAG-compliant until APCA is integrated into WCAG 3.0. WCAG 3.0 is still a work in progress and currently does not have an anticipated release date.
+
+## Visual themes
+
+GitHub offers two parent themes: 
+
+1. Day (light), and 
+2. Night (dark). 
+
+Each parent theme has a default appearance, and then a suite of sub-themes. Sub-themes include considerations for high and low contrast experiences, as well as certain vision conditions.
+
+<img src={ButtonFunctionalIcon} alt="GitHub's theme picker showing Day and Night parent themes, with swatches for each of the nine sub-themes. The Day theme is active. Cropped screenshot'" />
+
+It is possible to set GitHub to use a dark sub-theme when a parent Day theme has been set, and vice-versa. This allows a user to have a fine-grained control over their experience.
+
+### Default theme
+
+GitHub uses: 
+
+- **The default Day (light) theme** when a user visits the site in a logged out state and their operating system is set to light mode.
+- **The default Night (dark) theme** when a user visits the site in a logged out state and their operating system is set to dark mode.
+- Whatever theme the user has set in their account settings. 
+
+When not logged in, users are unable to adjust the theme in the [Appearance settings](https://github.com/settings/appearance).
+
+Because of this, [the default Day **and** Night theme **must both** satisfy WCAG criteria for color-related concerns](https://github.com/github/accessibility/blob/main/docs/coaching-recommendations/primer/color-modes.md#guidance). From a high-level, this means that **all** GitHub content displayed using the default Day and Night theme **must**:
+
+- Pass color contrast requirements, and
+- Have the mechanism for accessing and changing the theme be fully WCAG compliant.
+
+This is the strongest guarantee we have that: 
+
+1. Content can be perceived—and therefore understood and used—across a wide range of vision conditions, and that
+2. The mechanism to adjust GitHub's theme can be operated by the widest range of input modes.
+
+This allows users to use GitHub in a logged-out state. If a user is logged in, it also enables them to address their own specific usability and access needs, as well as aesthetic desires.
+
+### Non-default themes
+
+[The Night (dark) and **all** Day and Night sub-themes **do not** need to meet WCAG color contrast requirements](https://github.com/github/accessibility/blob/main/docs/coaching-recommendations/primer/color-modes.md#guidance), provided the default Day theme and its content is accessible. This guidance is derived from [WCAG 2.1 Technique G174](https://www.w3.org/WAI/WCAG21/Techniques/general/G174).
```

Comment:
Wait, doesn't the Night theme need to hit contrast? 😅

Reply:
I also don't think this makes sense. This means:

If a user is using dark mode on their mac and visit github.com (without being logged in) they get the dark theme that is not accessible and can't change it.